_includes/partials/contributor.html

```diff
@@ -1,62 +1,72 @@
-<div class="contributor flex"> 
-    <div class="container {% if page.full_width %}max-container{% endif %}">
-        <h2>Community Members</h2>
-        <p>A warm and welcoming collection of open sourcers.</p>
-
-        <ul class="contributor">  
-        <!-- List all Contributors -->
-        {% if "community.html" == page.name %}
-            {% for page in site.contributors %}
-                {% if page.layout != 'archive' %}
-                <li>
-                    <div class="square-image">
-                        <a href="{{page.url}}">
-                            <img src="{% include relative-src.html src=page.image_path %}" alt="{{ page.name }}"/>
-                        </a>
-                    </div>
-                    <div class="names">{{ page.name }}</a></div>
-
-                    {% if page.twitter %}
-
-                        <div class="twitter"><img src="{% include relative-src.html src='../images/social/twitter.png'%}"> 
-                            <a target="_blank" href="https://twitter.com/{{ page.twitter }}">{{ page.twitter }}</a></div>
-
-                    {% endif %}
-                    {% if page.github %}
-                        
-                        <div class="github"><img src="{% include relative-src.html src='../images/social/github.png'%}"> 
-                            <a target="_blank" href="https://github.com/{{ page.github }}">{{ page.github }}</a></div>
+<!DOCTYPE html>
+<html>
+    <head>
 
+    </head>
+    <body>
+        <div class="contributor flex"> 
+            <div class="container {% if page.full_width %}max-container{% endif %}">
+                <h2>Community Members</h2>
+                <p>A warm and welcoming collection of open sourcers.</p>
+                <h4 style="text-align:center;color:aliceblue;">
+                    <input style="padding-bottom:10px;padding-top:10px;width:300px; margin-left: auto; margin-right: auto;" type="submit" value="Join the Community" name="subscribe" id="mc-embedded-subscribe"></h4></span>
```

Comment:
The button visually behaves well, but does not link the user anywhere. This should navigate the user to the community Slack.